src/plugins/core/plugin_minify_html_literals.ts

```diff
@@ -0,0 +1,33 @@
+import { minifyHTMLLiterals } from 'minify-html-literals';
```

Comment:
don't import it here ;_)
do it like this: 
https://github.com/fuse-box/fuse-box/blob/master/src/plugins/core/plugin_less.ts#L11-L17

cuz that package is in dev dependencies it will simply fail

Reply:
ok.
just do `require('minify-html-literals')` before I use it then ?